### DISCUSSION TITLE
Copilot stopped working although I am an eligible student

### DISCUSSION THREAD

**Discussion by Haidar0096** [community/community#47544] [Copilot Conversations]
### Select Topic Area

Bug

### What should I do to regain access?

![image](https://user-images.githubusercontent.com/49842031/219013621-8340adcf-8c34-4463-8d21-c4df75f7572b.png)
![image](https://user-images.githubusercontent.com/49842031/219013722-1b49ed81-7c12-4c63-bcb5-98b76d8298dc.png)
![image](https://user-images.githubusercontent.com/49842031/219015311-83119b0a-3765-4112-b3fd-927455223a45.png)

![image](https://user-images.githubusercontent.com/49842031/219013485-03893aca-52cd-4976-b343-4ee6e2345c90.png)

**Comment by MarcHagen** [comment]
Maybe contact through support : https://support.github.com/request?tags=dotcom-direct
Or search for other issues: https://github.com/orgs/community/discussions/41371

**Reply by Haidar0096** [reply]
@MarcHagen thx, I already searched alot but people there only say it was resolved but they don't say how. Do you know if copilot is no longer free for students?

**Reply by MarcHagen** [reply]
Seems like a GitHub hickup.
If you are still a legit student it will take a bit. GitHub now checks on a monthly basis is this is still correct.

https://github.com/orgs/community/discussions/43634#discussioncomment-4629351

**Reply by Haidar0096** [reply]
> Seems like a GitHub hickup. If you are still a legit student it will take a bit. GitHub now checks on a monthly basis is this is still correct.
> 
> https://github.com/orgs/community/discussions/43634#discussioncomment-4629351

@MarcHagen so I should do nothing but wait and it will be resolved by itself?

**Comment by SilasRodrigues19** [comment]
Hey @Haidar0096 

I had the same problem and in my case I solved it as follows:

- I reinstalled the extension in VsCode.

- I clicked on VSCode's `Accounts` option and logged out with my GitHub account.

- I logged back into VSCode with the GitHub account and re-synced Copilot, then it was working again.

**Reply by Haidar0096** [reply]
@SilasRodrigues19 thanks for your suggestion, but this didn't work, and vscode has nothing to do with it, the access to copilot is revoked by github, see this screenshot from my billing page:
![image](https://user-images.githubusercontent.com/49842031/219020897-d1a52628-bcd7-4359-a6f8-432ec6a40be8.png)
![image](https://user-images.githubusercontent.com/49842031/219020960-ec94af13-174d-4984-be87-0c8193389850.png)

**Comment by Akash1134** [comment]
Hi there,

Are you still experiencing this issue? If yes, what is the exact error message you're seeing on your IDE? 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well. We know this causes a lot of inconvenience and we're sorry about that.

Here are a couple of steps to self-troubleshoot the issue:

-  If you are a current student, please ensure your student coupon is active on your GitHub account then when you visit https://github.com/github-copilot/signup the student subscription will be available.
- Please reverify yourself as a student, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."
- If that does not help, you will need to cancel your trial subscription, and then go to https://github.com/github-copilot/free_signup to claim your student subscription.

Let me know how it goes. Thanks!

**Comment by ghost** [comment]
I have this problem too if someone can help.
![image](https://github.com/community/community/assets/118917112/462e5a50-fc4f-4ffd-84de-7376e7fc190c)
but when I try to enter the copilot page it doesn't give me the free option.
It worked for me in the past in vs code and it just stopped working for some reason.
That is the output in VS:
![image](https://github.com/community/community/assets/118917112/ee032991-c115-49bb-82c6-1ed37288632b)
